src/pages/reference/accounts.md

```diff
@@ -68,12 +68,21 @@ You can see your referral invite statuses by navigating to the account page to:
 
 Every Account gets $5 of included credit per month. No credit card required.
 
-Users who upgrade to the Developer plan get $10's worth of credit each month and are billed for any usage above $10.
+Users who upgrade to the Developer plan do not get a time limit execution on their workloads compared to users on the starter plan.
```

Comment:
For benefits, don't say what they DONT get, say what they do

"unlimited execution"